app/common.py

```diff
@@ -54,3 +57,26 @@ class BadDataError(Exception):
 
     def __init__(self, message):
         super().__init__(message)
+
+
+def decode_token(token):
+    """
+    Decodes a JWT token using Firebase
+
+    Args:
+        token: A string of the encoded JWT
+
+    Returns:
+        dict: A dictionary of the key-value pairs from the decoded JWT
+
+    Raises:
+        Unauthorized: if the token is invalid or expired
+        ValueError: if `token` is not a string or is empty
+    """
```

Comment:
I feel Google's pydoc format is overly verbose and long, it's like 10+ lines for a simple function. I'd like to use the reST style (https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats) style throughout the project since it's the most concise and most widely used.

Reply:
I honestly just picked Google because it looks nice and clean in VSCode when hovering over functions. I'm open to using Rest style moving forward though, since it would probably make things a bit easier.